src/tink/sql/expr/Functions.hx

```diff
@@ -17,6 +17,9 @@ class Functions {
   public static function min<D,O>(e:Field<D,O>):Expr<D> 
     return ECall('MIN', cast [e]);
 
+  public static function trim<D,O>(e:Field<D,O>):Expr<String>
```

Comment:
I think this should be `public static function trim(e:Expr<String>):Expr<String>`